routes/posts.js

```diff
@@ -3,6 +3,14 @@ module.exports = function () {
     var express = require('express');
     var router = express.Router();
 
+    function require_signin (req, res, next) {

```

Comment:
You should abstract this function out into a helper functions directory.

Reply:
ok will perform in the coming sprint